src/pages/main/MainPage.tsx

```diff
@@ -110,10 +110,9 @@ export default function MainPage() {
   }, [cmdkReferenceJumpstart]);
 
   // reload project after local file changes
-  const debouncedCurrentProjectReload = useCallback(
-    debounce(triggerCurrentProjectReload, CodeViewSyncDelay),
-    [triggerCurrentProjectReload, fileTree, currentFileUid],
-  );
+  const debouncedCurrentProjectReload = useCallback(() => {
+    if (autoSave) debounce(triggerCurrentProjectReload, CodeViewSyncDelay);
```

Comment:
that function not work without adding call brackets
```suggestion
    if (autoSave) debounce(triggerCurrentProjectReload, CodeViewSyncDelay)();
```